src/sentry/relay/config/__init__.py

```diff
@@ -369,16 +379,16 @@ def get_at_path(self, *args):
 
         return None  # property not set or path goes beyond the Config defined valid path
 
-    def __get_data(self):
+    def __get_data(self) -> Any:
```

Comment:
this return type ought to be mapping.

not mutablemapping since this is allegedly an immutable class